CMakeLists.txt

```diff
@@ -551,6 +551,26 @@ else()
 		INSTALL_COMMAND ${CMD_MAKE} install)
 endif()
 
+option(USE_BUNDLED_INJA "Enable building of the bundled inja" ${USE_BUNDLED_DEPS})
```

Comment:
Could you please format this new chunk of code according to the `.cmake-format` in the root?